docs/Microservices/RPC/commandsRPC.md

```diff
@@ -0,0 +1,87 @@
+
+RPC is primarily used for communicating with the Remote Provision Server (RPS) for activating and/or deactivating AMT devices. Additional optional arguments allow for things such as easier development testing or for use in network environments utilizing proxies.
```

Comment:
Additional optional arguments. Two words that are not the same, but the optional cancels out the additional in a way. I'd say get rid of it.